src/AndroidDebugLauncher/InstallPaths.cs

```diff
@@ -121,6 +117,14 @@ out matchedPath
                 }
             }
 
+            IEnumerable<INDKFilePath> gdbServerPath = NDKPrebuiltFilePath.GDBServerPaths(targetArchitectureName);
+            result.GDBServerPath = GetNDKFilePath(
+                string.Concat("Android-", targetArchitectureName, "-GDBServerPath"),
```

Comment:
I couldn't find where this registry key is being set for `Android-<arch>-GDBPath`. I added the `Android-<arch>-GDBServerPath` query here, but I doubt this is being set anywhere.